CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 Records breaking changes from major version bumps
 
+## 3.0.0
+
+PR: [#21](https://github.com/alphagov/digitalmarketplace-docker-base/pull/21)
+
+### What changed?
+
+We're now using yarn instead of bower and npm. Also updates the base image we build from.
```

Comment:
It might be good to flesh this out a bit. Could add some info as to what version of yarn we're installing, how we're installing it, what we had to update the base version from and too, and how that affects the os version that we'll now be running?